lib/param_parsing/toml.go

```diff
@@ -0,0 +1,39 @@
+package param_parsing
+
+import (
+	"github.com/spf13/viper"
+	"github.com/warrensbox/terraform-switcher/lib"
+	"os"
+)
+
+const tfSwitchTOMLFileName = ".tfswitch.toml"
+
+// getParamsTOML parses everything in the toml file, return required version and bin path
+func getParamsTOML(params Params) Params {
+	tomlPath := params.ChDirPath + "/" + tfSwitchTOMLFileName
+	if tomlFileExists(params) {
+		logger.Infof("Reading configuration from %s", tomlPath)
+		configfileName := lib.GetFileName(tfSwitchTOMLFileName)
+		viperParser := viper.New()
+		viperParser.SetConfigType("toml")
+		viperParser.SetConfigName(configfileName)
+		viperParser.AddConfigPath(params.ChDirPath)
+
+		errs := viperParser.ReadInConfig() // Find and read the config file
+		if errs != nil {
+			logger.Fatalf("Unable to read %s provided", tomlPath)
+			os.Exit(1)
+		}
+
+		params.Version = viperParser.GetString("version") // Attempt to get the version if it's provided in the toml
+		params.CustomBinaryPath = viperParser.GetString("bin")
+	} else {
+		logger.Infof("No configuration file at %s", tomlPath)
```

Comment:
Do we really need to inform on non-existing TOML file? 🤔 I'm not sure we do.

Reply:
This should be debug log. I will change that

Reply:
I seem to think users have no option to configure logging level and hence debug messages are printed always. I see no sense in printing this info as this info makes no value for user. IMHO.

Reply:
You are probably right. Will fix that.